instagram/src/components/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import './SearchBar.css';
+import person from './person.jpg';
```

Comment:
Great job getting all the data to appear through image imports! Hopefully after today you'll be able to use font awesome